master_server/dummy_vm_server.py

```diff
@@ -32,38 +33,43 @@ def run(self, host=None, port=None, debug=None, load_dotenv=True, **options):
 MASTER_SERVER = 'http://127.0.0.1:5000'
 VM_ADDRESS = 'http://127.0.0.1:'
 request_id = ''
-task_response = Request_pb2.TaskResponse()
+#task_response = Request_pb2.TaskResponse()
```

Comment:
if we don't need this line we should just remove it instead of commenting, same for many commented lines below

Reply:
Changes has been made as suggested.